argon2/src/block.rs

```diff
@@ -144,3 +197,32 @@ impl Zeroize for Block {
         self.0.zeroize();
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    #[cfg(target_arch = "x86_64")]
+    #[test]
+    fn compress_avx2() {
+        let lhs = Block([
+            0, 0, 0, 2048, 4, 2, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
```

Comment:
It would be more concise to initialize block with zeros and then `copy_from_slice` the first 7 values.